test/unit/commands_test.rb

```diff
@@ -194,3 +194,87 @@ class TestList < HammerCLIForeman::ListCommand
   end
 
 end
+
+describe HammerCLIForeman::ListCommand do
+  class TestList < HammerCLIForeman::ListCommand
+    resource :domains
+    option '--page', 'PAGE', ''
+    option '--per-page', 'PER_PAGE', ''
+  end
+
+  def build_items(cnt)
+    (1..cnt).map do |i|
+      {:id => i, :name => "Item #{i}"}
+    end
+  end
+
+  def expect_paged_call(page, per_page, item_cnt)
+    api_expects(:domains, :index, "List records page #{page}") do |par|
```

Comment:
All of our other tests which hit the stubbed API are considered functional tests because they test behavior with respect to the API. Usually unit tests test a single method without exercising any of its interfaces. Could you please justify why you've placed this in the unit tests directory?